litmus-portal/backend/graphql-server/pkg/database/mongodb/project.go

```diff
@@ -0,0 +1,38 @@
+package database
+
+import (
+	"context"
+	"log"
+
+	"github.com/litmuschaos/litmus/litmus-portal/backend/graphql-server/graph/model"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+var projectCollection *mongo.Collection
+
+// CreateProject ...
+func CreateProject(ctx context.Context, project *model.Project) error {
+	// ctx, _ := context.WithTimeout(backgroundContext, 10*time.Second)
+	_, err := projectCollection.InsertOne(ctx, project)
+	if err != nil {
+		log.Print("Error creating Project: ", err)
+		return err
+	}
+
+	return nil
+}
+
+//GetProject ...
+func GetProject(ctx context.Context, projectID string) (*model.Project, error) {
+	// ctx, _ := context.WithTimeout(backgroundContext, 10*time.Second)
```

Comment:
It will be good to have this context timeout in all the places

Reply:
Yes but handling database with the same context as the request will be better as it will be having the same Timeout and all the things. So the UI request can be made aware if the resource has actually been created or edited in the database. Else the database operations will take place in  background irrespective of the UI request

Reply:
I have kept that commented if in future we need some different context

Reply:
Okay